README.md

```diff
@@ -1 +1,152 @@
-# Recipe-app
\ No newline at end of file
+# Recipe App
+
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contribution](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Recipe App <a name="about-project">
+
+Hi there! Welcome to Recipe App</a>
+
+A recipe web app is an online platform that provides users with a wide range of recipes for different types of meals. It also includes a food list and recipe list that display items added by the logged-in user. Users can add new food and recipes, and owners of recipes can delete them. The app also includes a public recipe list that displays all public recipes ordered by the latest, as well as a shopping list view that shows missing ingredients for all recipes of the logged-in user
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+<summary>Server</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Shows list of foods and recipe**
+- **Authentication and Authorization**
+- **General shoping list**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need to install the following:
+
+- Ruby
+- PostgreSQL
+- Rails gem
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```snippet
+  git clone https://github.com/UbdaNam/Recipe-app.git
+```
+
+### Install
+
+```snippet
+  bundle install
+```
+
+Install this project with: Any code editor or terminal which you are comfortable with
+
+### Usage
+
+In the project directory, you can run:
+
+```Database
+rails db:create
+```
+
+```Migrate db
+rails db:migrate
+```
+
+```Start server
+rails s
+```
+
+<!-- AUTHORS -->
+
+## 👥 Author <a name="authors"></a>
+
+👤 **Ubda Nam**
+
+- GitHub: [@UbdaNam](https://github.com/UbdaNam)
+- LinkedIn: [Abdurahim M.](https://www.linkedin.com/in/abdurahim-miftah/)
+
+👤 **Wullah Jonas Nuta**
+
+- GitHub: [@jonas-45](https://github.com/jonas-45)
+- Twitter: [@jonaswullah](https://twitter.com/jonaswullah)
```

Comment:
- [ ] Either fix this link or remove it as it points to a profile that does not exist, for a more professional readme file.